contracts/contracts/strategies/NativeStaking/ValidatorAccountant.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { ValidatorRegistrator } from "./ValidatorRegistrator.sol";
+import { IWETH9 } from "../../interfaces/IWETH9.sol";
+
+/// @title Validator Accountant
+/// @notice Attributes the ETH swept from beacon chain validators to this strategy contract
+/// as either full or partial withdrawals. Partial withdrawals being consensus rewards.
+/// Full withdrawals are from exited validators.
+/// @author Origin Protocol Inc
+abstract contract ValidatorAccountant is ValidatorRegistrator {
+    /// @notice The maximum amount of ETH that can be staked by a validator
+    /// @dev this can change in the future with EIP-7251, Increase the MAX_EFFECTIVE_BALANCE
+    uint256 public constant MAX_STAKE = 32 ether;
+
+    /// @notice Keeps track of the total consensus rewards swept from the beacon chain
+    uint256 public consensusRewards = 0;
```

Comment:
In Solidity, variables are automatically initialized to zero. Therefore, explicitly initializing variables to zero is not mandatory and can be skipped.

Reply:
Good point, fixed here: https://github.com/OriginProtocol/origin-dollar/pull/2053/commits/8bf9a268ffc433b95f79ef5b6c15cb9d19a9cc6d